packages/perseus/src/widgets/label-image/label-image.tsx

```diff
@@ -66,6 +66,7 @@ type Point = {
     y: number;
 };
 
+// TODO: should this be using WidgetProps / PerseusLabelImageWidgetOptions?
```

Comment:
Yes.  Or... we use this approach for all widget `RenderProps` types where we explicitly define each key. I think it makes this more verbose but potentially clearer. @benchristel I think you've had thoughts about this before, haven't you? 

Although not part of this PR, this should also be named `RenderProps` for consistency.